### PR TITLE
fixing issue #7011

### PR DESCRIPTION
**Pull request by hkernbach**
fixing https://github.com/arangodb/arangodb/issues/7011

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js [hunk 1]
```diff
@@ -1208,9 +1208,9 @@
         tableContent.push(
           window.modalView.createCheckboxEntry(
             'new-app-replace',
-            'Keep configuration and dependency files?',
+            'Discard configuration and dependency files?',
             true,
-            "Should this app's configuration be saved before replacing the app?",
+            "Should this app's configuration be removed before replacing the app?",
```
Comment (pluma) [inline js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js]: I'd suggest something like this instead:

> Discard existing configuration and dependencies?

> Should this service's existing configuration and settings be removed completely before replacing the service?

"Dependency files" aren't really a thing. When the flag is set, the existing service will be purged completely before the new service is installed. That includes metadata like config and dependencies.

Also I'm not sure where else we need to change this but I'm pretty sure we switched to calling them "services" at some point throughout 2.x and should avoid/change any occurrences of the phrase "app" or "application".

Comment (pluma) [reply js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js]: (changing all occurrences of "app" or "application" may be out of scope for this PR, maybe @ifcologne can decide)

Comment (hkernbach) [reply js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js]: Applied your requested change.

Comment (hkernbach) [reply js/apps/system/_admin/aardvark/APP/frontend/js/arango/arango.js]: > (changing all occurrences of "app" or "application" may be out of scope for this PR, maybe @ifcologne can decide)

Yes it is, We can to that in another PR.

**Comment by hkernbach** [comment]
@pluma mergeable then?

**Pull request merged**